coffee/src/CoffeeMachineSimulation.java

```diff
@@ -0,0 +1,212 @@
+import java.util.Scanner;
+
+public class CoffeeMachineSimulation {
+
+    private enum State {
+        Ready,
+        Off,
+        SelectCoffee,
+        FillWater,
+        FillCoffee,
+        FillMilk,
+        FillCups
+    }
+    private Scanner scan = new Scanner(System.in);
+
+    private final int ESPRESSO = 0;

```

Comment:
It can be an enum too

Reply:
I've created enum for coffee types and made the appropriate changes in the code.